### DISCUSSION TITLE
Some Qs

### DISCUSSION THREAD

**Discussion by cloudcompute** [featureform/featureform#698] [Q&A] [answered]
Hi

I have few questions relating to FF

a. Which companies are using it in production?

b. Embedding Hub is in Alpha right now. Can we use a vector search engine like Vespa in place of it?

c. Is there a comparison page that list the differences between Open Source and Enterprise editions?

Thanks

**Comment by simba-git** [comment, marked as answer]
a) Happy to provide you with some case studies, could you email Shab, shab (at) featureform.com

b) We plan to have support for Vespa and other vector databases like Weaviate & Redis in May

c) We're in the midst of a website revamp, the main differences between OSS & Enterprise:
1. Access control & Governance functionality
2. Custom feature lifecycles (like dev cluster vs prod cluster separation)
3. Support & SLAs

Note that Enterprise Featureform is not SaaS, it runs on-prem or in your cloud
